src/content/articles/en/developing-meeting-culture-employees-love.mdx

```diff
@@ -0,0 +1,107 @@
+Ask yourself who the bare minimum of people are to get that job done and only invite them. The fewer people attend a meeting the less time you spend on it.
+A 1-hour meeting with 48 people is 48 hours of time not only one hour.
+
+#### Nobody has to attend the meeting
+
+Even with the minimum amount of people attending, make sure that those who couldn’t make it do not block you from having this meeting.
+That also means that you have the meetings only to discuss things, not to decide. The decision will be based on the outcomes of that meeting but will be made after the meeting. That will ensure that the people who missed out have a chance to get up-to-date and be part of the decision.
+
+### 4. No meeting without a desired outcome and agenda
+
+When you plan a meeting, ask yourself “Why do I need this meeting What do I want to achieve with it?”
+That desired outcome will define the topic of your meeting and based on that you can create an agenda for the meeting things that need to be discussed.
+You should have these two prepared before you send out invites because the outcome and agenda will be the information upon which the attendees can decide if they need to be in the meeting.
+The agenda will also be the base for everybody attending to prepare the meeting.
+
+### 5. Every meeting has a fixed duration
+
+Did you ever plan a meeting and wondered how long that meeting should take place? Well, now you won’t anymore.
+The length of each meeting can be calculated by the following formula:
+
+<Calculation># attendees x 10 minutes + 15 minutes</Calculation>
+
+The 10 minutes and 15 minutes can be adapted to your company but I found that those numbers are pretty much suitable for most of the companies.
+
+#### Each attendee has a speaking slot
+
+The first part of the formula foresees a speaking slot of 10 minutes for everyone attending. During that time nobody else speaks but the one in turn.
+
+#### Q + A
+
+The last 15 minutes a reserve for a Q + A with "A" being "answers" and "actions points"
+During that time the initiator of the meeting takes the role of the moderator.
+If you need longer than this time for your meeting that means your agenda is too big or people didn’t prepare property (see following section).
+
+### 6. Everybody prepares for and reviews the meeting
+
+Once every attendee has accepted their invite to the meeting they are required to prepare for the meeting. Do the research, note down your ideas or questions.
+If there is something that you want others to check upfront, make sure that you communicate this early enough. You won’t have time to show a 20-minute video during the meeting.
+After the meeting has taken place, the initiator collects everything that was discussed and provides the meeting notes to the knowledge base.
+This can be made easier if you record meetings. but videos should never replace written notes because you need more time to consume videos than notes.
```

Comment:
```suggestion
This can be made easier if you record meetings, but videos should never replace written notes because you need more time to consume videos than notes.
```